weave/trace_server/migrations/002_feedback.up.sql

```diff
@@ -0,0 +1,49 @@
+
+CREATE TABLE feedback (
+    /*
+    `id`: The unique identifier for the feedback. This is a UUID.
+    */
+    id String,
+
+    /*
+    `project_id`: The project identifier for the call. This is an internal
+    identifier that matches the project identifier in the W&B API.
+    It is stored for feedback to allow efficient permissions filtering.
+    */
+    project_id String,
+
+    /*
+    `call_id`: The unique identifier for the call. This is typically a UUID.
+    */
+    call_id String,
```

Comment:
If we wanted to support "refs" we could modify this to `target_id`, then have a `kind` field (call, object, table), then have an "extra" field. This would permit us to lookup via the ref parts

Reply:
I'm proposing we store the ref string, allowing us to do a prefix based query to get all of the feedback on e.g. a dataset or its rows.